src/table/table.scss

```diff
@@ -122,265 +251,316 @@
         }
       }
 
-      &.iui-tables-head {
-        @include themed {
-          background-color: t(iui-color-background-3);
-        }
-        user-select: none;
+      // #region Indented rows
+      &.iui-expander-indent-0 {
+        padding-left: 0;
+      }
 
-        .iui-icon {
-          @include themed {
-            fill: t(iui-icons-color-actionable);
-          }
+      &.iui-expander-indent-1 {
+        padding-left: $iui-xl;
+      }
 
-          &:hover {
-            @include themed {
-              fill: t(iui-icons-color-actionable-hover);
-            }
-          }
-        }
+      &.iui-expander-indent-2 {
+        padding-left: $iui-xl * 2;
+      }
 
-        > .iui-filter {
-          margin-left: $iui-sm;
-          cursor: pointer;
+      &.iui-indent-1 {
+        padding-left: $iui-xl + $iui-s;
+      }
 
-          > .iui-icon {
-            width: $iui-icons-default;
-            height: $iui-icons-default;
-          }
-        }
+      &.iui-indent-2 {
+        padding-left: ($iui-xl + $iui-s) * 2;
+      }
+      // #endregion Indented rows
+    }
 
-        > .iui-sort {
-          flex-grow: 1;
-          justify-content: flex-end !important;
+    // #region Selection
+    &:not(.iui-selected) + .iui-selected {
+      padding-bottom: $iui-xxs;
+      border-bottom: none;
+    }
 
-          > .iui-icon-wrapper {
-            cursor: pointer;
+    &.iui-selected {
+      padding: 0;
+      border-bottom: none;
+      @include themed {
+        border: solid 1px t(iui-color-foreground-primary);
+        background: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
+      }
 
-            > .iui-icon {
-              width: $iui-icons-default;
-              height: $iui-icons-default;
-            }
-          }
+      &:hover {
+        @include themed {
+          background: rgba(
+            t(iui-color-foreground-primary-rgb),
+            t(iui-opacity-5)
+          );
         }
+      }
 
-        > .iui-filter,
-        > .iui-sort,
-        > .iui-sort > .iui-icon-wrapper {
-          height: $iui-icons-default;
-          width: $iui-icons-default;
-          display: flex;
-          justify-content: center;
-          align-items: center;
-          visibility: hidden;
+      + .iui-selected {
+        border-bottom: none;
+        padding-bottom: $iui-xxs;
+        padding-top: 0;
```

Comment:
Do these rules do anything? I tried to turn them off and leave only border and padding bottom, did not change anything..

Reply:
I dont think padding-top here is needed, cause I could not find the place, where it was set to something more...

Reply:
Your right, padding is correct because of line 190 with .iui-selection { padding: 0; } probably this is the early solution for fixing budging content and later on I found out a better solution. 
You can remove padding-top in this rule.